scripts/tools/check_includes_config.py

```diff
@@ -164,8 +164,10 @@
     'src/lib/dnssd/minimal_mdns/ResponseSender.h': {'list'},
 
     # Not really for embedded consumers; uses std::deque to keep track
-    # of a list of discovered things.
-    'src/controller/SetUpCodePairer.h': {'deque'},
+    # of a list of discovered things and std::vector to keep track of
```

Comment:
maybe consider not explaining why we need these (that couples usage and is bound to become out of date if people just shuffle things around. It seems just saying "small embedded devices are not commissioners" or some variant of that may be sufficient.

Reply:
OK, can do.